src/GraphQL.Tests/Validation/InputFieldsAndArgumentsOfCorrectLengthRequiredTests.cs

```diff
@@ -0,0 +1,358 @@
+using GraphQL.Validation.Rules;
+
+namespace GraphQL.Tests.Validation;
+
+public class InputFieldsAndArgumentsOfCorrectLengthRequiredTests : ValidationTestBase<InputFieldsAndArgumentsOfCorrectLength, ValidationSchema>
```

Comment:
You don't want to put tests in InputFieldsAndArgumentsOfCorrectLengthTests.cs?

Reply:
Either way is fine with me.  It just seemed easier at the time to copy the whole file.  Go ahead and make any changes to the PR you wish.

Reply:
It seemed to me we had tests and I really found a file with tests. The next question that I asked myself - how they differ? Only `non-null` type modifier?

Reply:
Right; the only difference is that it is testing fields with non-null modifiers.  The tests are all the same, I think.

Reply:
ok